### DISCUSSION TITLE
Starchat-alpha

### DISCUSSION THREAD

**Discussion by da3dsoul** [oobabooga/text-generation-webui#2029] [General]
https://huggingface.co/HuggingFaceH4/starchat-alpha

Has anyone tried this? I got it running with no effort, but it's slow. I thought it was neat, as generalized models like vicuna are pretty good at code, but the lower context of 2048 is hard for coding.

**Comment by Kwisss** [comment]
Yeah I tried it, But I don't have the gpu memory, so it's slow indeed.
Maybe that a ggml version can be as quick as other ggml versions

**Reply by da3dsoul** [reply]
Yeah I managed to load it in 14GB without issue, but it's still like 3s/it

**Comment by localagi** [comment]
Anyone can tell me the model params? Are you loadin safetensors?

**Reply by da3dsoul** [reply]
I didn't use anything special. Namely, because it's not special, there's no quantization parameters
